src/nbl/CMakeLists.txt

```diff
@@ -650,8 +650,13 @@ set(_NBL_DEFINE_FILE_WRAPPER_ ${CMAKE_CURRENT_BINARY_DIR}/include/define.h)
 write_source_definitions("${_NBL_DEFINE_FILE_WRAPPER_}" "${_NBL_SOURCE_DEFINITIONS_}")
 
 #on MSVC it won't compile without this option!
+option(NBL_TREAT_WARNINGS_AS_ERRORS "Treat warnings as errors" OFF)
```

Comment:
also move this option and [`NBL_SILENT_WARNINGS_WITH_WRAPPER`](https://github.com/Devsh-Graphics-Programming/Nabla/pull/687/files#r1584153583) a little bit higher maybe [here](https://github.com/Devsh-Graphics-Programming/Nabla/blob/master/src/nbl/CMakeLists.txt#L83)

Reply:
+ both options enabled by default